scripts/inner/kubestellar-kube-bind

```diff
@@ -138,10 +181,22 @@ else
     echo "consumer side secret is $consumer_secret_name"
 fi
 
-KUBECONFIG=$space_conf_file
+if fgrep -x "${resource_to_bind}" "$request_file"; then
+    echo "The request file has junk in it; see?" >&2
+    cat "$request_file" >&2
+    mv "$request_file" "$request_file".junk
+    fgrep -x -v "${resource_to_bind}" "$request_file".junk > "$request_file"
+    echo Removed >&2
+fi
```

Comment:
I added this because I got the following failure from `kubestellar-kube-bind` when testing on my laptop.

```
...
++ kubectl bind apiservice --remote-kubeconfig-namespace kube-bind --remote-kubeconfig-name kubeconfig-ms6gs --skip-konnector -f konnector-imw1/locations-request.yaml
+ sub_result='DISCLAIMER: This is a prototype. It will change in incompatible ways at any time.

Error: failed to unmarshal manifest: error converting YAML to JSON: yaml: line 2: mapping values are not allowed in this context'
+ echo 'Error occurred during binding locations for imw1'
Error occurred during binding locations for imw1
+ echo 'Result of kubectl bind apiservice: DISCLAIMER: This is a prototype. It will change in incompatible ways at any time.

Error: failed to unmarshal manifest: error converting YAML to JSON: yaml: line 2: mapping values are not allowed in this context'
Result of kubectl bind apiservice: DISCLAIMER: This is a prototype. It will change in incompatible ways at any time.

Error: failed to unmarshal manifest: error converting YAML to JSON: yaml: line 2: mapping values are not allowed in this context
+ echo 'Request file:'
Request file:
+ cat konnector-imw1/locations-request.yaml
locations
apiVersion: kube-bind.io/v1alpha1
kind: APIServiceExportRequest
metadata:
  name: locations.edge.kubestellar.io
spec:
  resources:
  - group: edge.kubestellar.io
    resource: locations
status: {}

+ echo
...
```